src/pybind/mgr/dashboard/tools.py

```diff
@@ -744,4 +745,4 @@ def getargspec(func):
 def str_to_bool(var):
```

Comment:
could you add a docstring like this?:
```python
"""
>>> str_to_bool('true') and str_to_bool('yes') and str_to_bool('1') 
True
"""
```
and make sure it's tested by tox?

Reply:
Done